src/coreclr/inc/yieldprocessornormalized.h

```diff
@@ -43,6 +43,8 @@ class YieldProcessorNormalization
         return s_isMeasurementScheduled;
     }
 
+    static void GCHeapSetYieldProcessorScalingFactor();
```

Comment:
How about `NotifyGC()`? I figure the rest would be implied by the class name. Maybe there's a better name.